foxglove_bridge_base/include/foxglove_bridge/websocket_server.hpp

```diff
@@ -227,6 +230,9 @@ inline Server<ServerConfiguration>::Server(std::string name, LogCallback logger,
     std::bind(&Server::handleMessage, this, std::placeholders::_1, std::placeholders::_2));
   _server.set_reuse_addr(true);
   _server.set_listen_backlog(128);
+
+  // Callback queue for handling client requests.
+  _handlerCallbackQueue = std::make_unique<CallbackQueue>(_logger, 1ul /* 1 thread */);
```

Comment:
I've seen it done this way which I like since it uses the same parameter name.
```suggestion
  _handlerCallbackQueue = std::make_unique<CallbackQueue>(_logger, /*numThreads=*/ 1ul);
```